packages/remix-dev/cli/create.ts

```diff
@@ -451,3 +452,80 @@ async function detectTemplateType(
   }
   return "remoteTarball";
 }
+
+function untype(filename: string, source: string, projectDir: string): string {
+  let result = babel.transformSync(source, {
+    filename,
+    presets: [[babelPresetTypeScript, { jsx: "preserve" }]],
+    plugins: [babelPluginSyntaxJSX],
+    compact: false,
+    retainLines: true,
+    cwd: projectDir,
+  });
+
+  if (!result || !result.code) {
+    throw new Error("Could not parse typescript");
+  }
+
+  /*
+    Babel's `compact` and `retainLines` options are both bad at formatting code.
+    Use Prettier for nicer formatting.
+  */
+  return prettier.format(result.code, { parser: "babel-ts" });
```

Comment:
We shouldn't need `babel-ts` here now that it's JavaScript and not TypeScript, right?